extensions/reactive-datasource/deployment/src/main/java/io/quarkus/reactive/datasource/deployment/VertxPoolBuildItem.java

```diff
@@ -1,37 +0,0 @@
-package io.quarkus.reactive.datasource.deployment;
-
-import io.quarkus.builder.item.MultiBuildItem;
-import io.quarkus.runtime.RuntimeValue;
-import io.vertx.sqlclient.Pool;
-
-/**
- * A build item for Vertx {@link Pool}s.
- * <p>
- * If you inject this build item when recording runtime init template calls, you are guaranteed the Pool configuration
- * has been injected and Pools can be created.
- */
-public final class VertxPoolBuildItem extends MultiBuildItem {
```

Comment:
This is not used anywhere in a our code and there is no good reason why anyone else would use it either

Reply:
We could potentially keep the build item around just throw an exception if the getter methods are called.